lib/puppet/util/ssl.rb

```diff
@@ -50,4 +52,41 @@ def self.cn_from_subject(subject)
   def self.is_possibly_valid_dn?(dn)
     dn =~ /=/
   end
+
+  ##
+  # Extract and format meaningful error messages from OpenSSS::OpenSSLErrors
```

Comment:
OpenSSS 🐍  typo.

Reply:
I fixed this in my PR that I'm working on.